test/unit/collaborators.test.js

```diff
@@ -1,16 +1,120 @@
 'use strict';
 
+const assert = require('assert');
+
 const {
+  isCollaborator,
   getCollaborators
 } = require('../../lib/collaborators');
-const { readme, collaborators } = require('../fixtures/data');
+const { readme, readmeNoTsc, readmeNoTscE, readmeNoCollaborators, readmeNoCollaboratorE, readmeUnordered, collaborators } = require('../fixtures/data');
 const TestLogger = require('../fixtures/test_logger');
-const logger = new TestLogger();
-const assert = require('assert');
+const assertThrowsAsync = require('../fixtures/assert_throws_async');
 
 describe('collaborators', function() {
-  it('getCollaborators', async function() {
-    const parsed = await getCollaborators(readme, logger, 'nodejs', 'node');
-    assert.deepStrictEqual(parsed, collaborators);
+  const collaborator = collaborators.get('bar');
+  let logger = null;
+
+  beforeEach(() => {
+    logger = new TestLogger();
+  });
+
+  afterEach(() => {
+    logger.clear();
+  });
+
+  describe('Collaborator', () => {
+    describe('isActor', () => {
+      it('should return false if no actor is specified', () => {
+        assert.strictEqual(collaborator.isActor(), false);
```

Comment:
This should be `collaborator.isActor(null)`, github returns the actor as `null` if it's a ghost.